Extensions/TextObject/textruntimeobject-pixi-renderer.js

```diff
@@ -105,6 +105,10 @@ gdjs.TextRuntimeObjectPixiRenderer.prototype.getHeight = function() {
     return this._text.height;
 };
 
+gdjs.TextRuntimeObjectPixiRenderer.prototype.setWidth = function(newWidth) {};
+
+gdjs.TextRuntimeObjectPixiRenderer.prototype.setHeight = function(newHeight) {};
+
```

Comment:
Note to me : 
I can clean these lines, it's done [here now](https://github.com/4ian/GDevelop/commit/0d5fbdabc998a3673672dcbac8a6d4ade33a6cd0).